website/content/en/preview/concepts/settings.md

```diff
@@ -68,6 +68,9 @@ data:
   aws.interruptionQueueName: karpenter-cluster
   # Global tags are specified by including a JSON object of string to string from tag key to tag value
   aws.tags: '{"custom-tag1-key": "custom-tag-value", "custom-tag2-key": "custom-tag-value"}'
+  # Reserved ENIs are not included in the calculations for max-pods or kube-reserved
+  # This is most often used in the VPC CNI custom networking setup https://docs.aws.amazon.com/eks/latest/userguide/cni-custom-network.html
+  aws.reservedENIs: "1"
```

Comment:
As discussed offline, I can't think of a use-case where you'd need to adjust this in an AWSNodeTemplate. The VPC CNI custom networking configuration would be for all nodes in the cluster unless you've deployed multiple CNIs that select different groups of nodes which is possible, but probably not widely done in practice. I'm not strictly opposed though since I like the NodeTemplate API better than throwing things in Settings.

Reply:
We can always move it later if we decide to rip out the settings at a later date.